src/app/feed/info-box/info-box.component.spec.ts

```diff
@@ -22,7 +22,83 @@ describe('Component: InfoBox', () => {
 	});
 
 	it('should create an instance', async(() => {
-		const component = TestBed.createComponent(InfoBoxComponent);
+		const fixture = TestBed.createComponent(InfoBoxComponent);
+		const component = fixture.debugElement.componentInstance;
 		expect(component).toBeTruthy();
 	}));
+
+	it('should give valid result on non-empty ApiResponseResult', async(() => {
+		const fixture = TestBed.createComponent(InfoBoxComponent);
+		const component = fixture.debugElement.componentInstance;
+
+		component.ApiResponseResult = [
+			{
+				hashtags: [
+					'fossasia',
+					'opensource',
+					'pslab',
+					'35c3',
+					'madeinsingapore',
+					'fossasia',
```

Comment:
repeated?

Reply:
@simsausaurabh it is repeated because the function being called group them with the count of each hashtags. And then sort them in order of their count.